target/linux/ramips/dts/ESR600.dts

```diff
@@ -0,0 +1,211 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+/dts-v1/;
+
+#include "mt7620a.dtsi"
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+/ {
+	compatible = "engenius,esr600", "ralink,mt7620a-soc";
+	model = "EnGenius ESR600";
+
+	memory@0 {
+		device_type = "memory";
+		reg = <0x0 0x4000000>;
+	};
+
+	chosen {
+		bootargs = "console=ttyS0,115200";
+	};
+
+	aliases {
+		led-boot = &led_power;
+		led-failsafe = &led_power;
+		led-running = &led_power;
+		led-upgrade = &led_power;
+	};
+
+	leds {
+		// FIXME: locate WPS 5g blue LED
+		// ISSUE: taking the WAN LED on gpio2/5 
+		// causes AR8327 switch to fail initialization
+
+		compatible = "gpio-leds";
+
+		led_power: power {
+			label = "esr600:amber:power";
+			gpios = <&gpio0 9 GPIO_ACTIVE_LOW>;
+		};
+		wps2g {
+			label = "esr600:amber:wps2g";
+			gpios = <&gpio2 6 GPIO_ACTIVE_LOW>;
+		};
+		wlan5g {
+			label = "esr600:blue:wlan5g";
+			gpios = <&gpio0 7 GPIO_ACTIVE_LOW>;
+		};
+		wlan2g {
+			label = "esr600:blue:wlan2g";
+			gpios = <&gpio3 0 GPIO_ACTIVE_LOW>;
+		};
+	};
+
+	keys {
+		compatible = "gpio-keys";
+
+		reset {
+			label = "reset";
+			gpios = <&gpio0 1 GPIO_ACTIVE_LOW>;
+			linux,code = <KEY_RESTART>;
+			debounce-interval = <60>;
+		};
+
+		wps {
+			label = "wps";
+			gpios = <&gpio0 2 GPIO_ACTIVE_LOW>;
+			linux,code = <KEY_WPS_BUTTON>;
+			debounce-interval = <60>;
+		};
+
+	};
+
+ };
+
+&gpio1 {
+	status = "okay";
+};
+&gpio2 {
+	status = "okay";
+};
+&gpio3 {
+	status = "okay";
+};
+
+&spi0 {
+	status = "okay";
+
+	flash@0 {
+		compatible = "jedec,spi-nor";
+		reg = <0>;
+		spi-max-frequency = <50000000>;
+		m25p,fast-read;
+	
+		partitions {
+			compatible = "fixed-partitions";
+			#address-cells = <1>;
+			#size-cells = <1>;
+			   
+			partition@0 {
+				label = "u-boot";
+				reg = <0x0 0x30000>;
+				read-only;
+			};
+
+			partition@30000 {
+				label = "u-boot-env";
+				reg = <0x30000 0x10000>;
+				read-only;
+			};
+
+			factory: partition@40000 {
+				label = "factory";
+				reg = <0x40000 0x10000>;
+				read-only;
+			};
+
+			iNIC_rf: partition@50000 {
+				label = "iNIC_rf";
```

Comment:
Consider using lower case here and for the node name.

Reply:
I would have, but the "iNIC_rf" mixed case partition name was what the OEM was using, and the mixed case usage was present in the WN-AC1167GR, WN-AC733GR3, WN-GX300GR, and WN-AX1167GR DTSs, while it appears nowhere else in lowercase.